core/lib/utils/wrapper.dart

```diff
@@ -0,0 +1,4 @@
+class Wrapped<T> {
```

Comment:
Can you explain about this? Quite cool?

Reply:
@hoangdat this class uses send parameter is null in function copyWith